### PR TITLE
Define enumerations for animation

### PR DESCRIPTION
**Pull request by shoheiyokoyama**
#565 
into `development`

### CODE REVIEW

File: Sources/iOS/Animation.swift [hunk 1]
```diff
@@ -55,6 +55,35 @@ public func AnimationFillModeToValue(mode: AnimationFillMode) -> String {
 	}
 }
 
+@objc(AnimationTimingFunction)
+public enum AnimationTimingFunction: Int {
+    case liner
```
Comment (danieldahan) [inline Sources/iOS/Animation.swift]: I think you meant to write `linear`.

File: Sources/iOS/Animation.swift [hunk 2]
```diff
@@ -55,6 +55,35 @@ public func AnimationFillModeToValue(mode: AnimationFillMode) -> String {
...
+    case easeIn
+    case easeOut
+    case easeInEaseOut
+    case systemDefault
```
Comment (danieldahan) [inline Sources/iOS/Animation.swift]: this can be default

File: Sources/iOS/Animation.swift (hunk 1 above)
Comment (shoheiyokoyama) [inline Sources/iOS/Animation.swift]: Oh, I'm sorry :disappointed:

File: Sources/iOS/Animation.swift (hunk 2 above)
Comment (shoheiyokoyama) [inline Sources/iOS/Animation.swift]: Thanks for your advice!

**Comment by danieldahan** [comment]
Your PR inspired me to refactor the Animation class, thank you! 

https://github.com/CosmicMind/Material/commit/486cab299bb6109132a151d1cf451f76b70122f6

**Pull request merged**